### ISSUE TITLE
Logo for NX front-end framework

### ISSUE THREAD

**Issue by ghost** [g-harel/open-source-logos#1]
## NX Framework

> Project org (many repos): https://github.com/nx-js
   Main repo: https://github.com/nx-js/framework
   Home page: https://nx-framework.com/

### Background

NX is a next-gen front-end framework. It consists of small Web Components based core and a lot of focused feature modules tied together with a middleware system. The feature modules are written to be usable by other frameworks as well. As an example I just released a [React port](https://github.com/solkimicreb/react-easy-state) for my observable / state management module.

### Description

I would like to keep the green/white colors from the home page, but it could be bolder with some contrast too. The flagship feature is modularity so something that outlines that would be great. It is also real DOM based framework (no VDOM) with more focus on implementing old battle tested ideas right, so maybe some nostalgic tone could be used too (for example inspiration from the HTML logo).

### Time Constraints

No real constraints. As the project uses (unshimmable) Proxies it is really a next-gen framework, it won't be production ready for a while (I aim to release v1 in the fall). I was kind of silently working on the next major beta version, which will be ready in a few weeks. A logo would be awesome with that, but no need to rush it.

**Comment by g-harel** [comment]
@nx-hacker-news could I get your thoughts on these, please be honest and as specific as you can be
![image](https://cloud.githubusercontent.com/assets/9319710/26516537/be808e5c-4255-11e7-8260-c82b864a0acc.png)

**Comment by solkimicreb** [comment]
Generally I like it a lot and I think I prefer the second one. I like the less cartoonish look of it. Here are a few questions and notes:

- I would probably use 2 versions of it, the one you have above and a monochrome (probably pure white) version for colorful backgrounds. Could you make a white version of it? Do you think it would look good?

- I don't yet have a strong enough name to rely on the logo only. Probably I would also like to put a text somewhere near the logo (like 'NX Framework'). Could you make a version where you fit it in somewhere or would it be too crowded?

- This is more of a personal question: Which one do you prefer from the ones you made?

These are just some notes, I think they are generally awesome and I trust you with the outcome. I would gladly use it as it is now too. Thanks for your work and for being so quick!

Have a nice weekend (:

Edit: sorry for using another account for this comment. I should really tidy up my GitHub accounts.

**Comment by g-harel** [comment]
Yep I'll make multiple versions, I think the white might actually look better since it would make the overlap's shadows darker than the logo.

Do you have a font preference ?

I think I like the one on the left better, I'm a little partial to the cartoon style though 😛

**Comment by solkimicreb** [comment]
I use the [Raleway Google font](https://fonts.google.com/specimen/Raleway) on the home page, but for the logo it's up to you.

I asked your opinion, because I was not so sure either (: I played around a bit with the icons and the home page in the last hours.

After some thinking I think I have to small problems with the first one, but I like the design more.

- It looks nice as it is, but I couldn't fit it with the current color theme. I was thinking about changing the color theme to the logo, but it is too bright and vivid to look good on big surfaces. Could you keep the design of the first one and use the greens from the other one or maybe something bold that has more contrast with the greens (like purple or some red/magenta colors)?

- The second one has a bit more 'air' (white space) inside it, which makes it work better when it is small (used on menus, favicon, etc.). Can you widen up the spaces a bit?

So in summary: I like the first version with a bit more space inside and with the other's or more some contrasting colors and with a monochrome version and with 'NX Framework' text somewhere (thats a long list 😄 ).

Sorry for changing my mind on the go ): Just as before, these are my notes, but I am newbie in design so feel free to do as you will. I really like what you made (:

Edit: by white space I mean the little white triangles inside.

**Comment by g-harel** [comment]
I think Raleway works pretty well here, good pick!

For now I kept the same green, its pretty much the only color on the page so I thought it would be awkward to add another. I'll totally try the other colors if you want to see them.

![image](https://cloud.githubusercontent.com/assets/9319710/26524868/c09cf56a-4311-11e7-98e3-512251b4449a.png)

**Comment by solkimicreb** [comment]
I think it's perfect 👍 Checked it in different sizes and with different backgrounds and it fits super cool. So in the end I would use these two with text and without text (4 all together). Two final questions 🙂  

- Does the top one have a white or a transparent background? (I would go for white)

- General design question: Do you think I should add an accent color to the page for actions items? From just gut feeling what tone would you put if you would?

Thx for the logo, it is awesome! Let me know if you would like to have a link, your name or anything else as credit on the places I use it (:

**Comment by solkimicreb** [comment]
I just saw [goo](https://github.com/g-harel/goo) and [diip](https://github.com/g-harel/diip). I think we are trying to solve very similar issues with a similar mindset 😄 The framework seems like a nice one. 👍  I only had time to run through the readMe, but it is definitely matching with my taste. Blobs seem to be a nice concept.

**Comment by g-harel** [comment]
Awesome, I'm glad you like it!

I like the color scheme you have going on, so adding highlight colors might not be necessary. However, I was playing with some color combos and I found that yellow goes quite well with the green you are using.

I've added the four versions [here](https://github.com/g-harel/open-source-logos/tree/master/projects/NX%20Framework)

Yeah making frameworks is a lot of fun, but a lot of work too!

**Comment by solkimicreb** [comment]
Hi!

Could you export the four final results as svg-s too? It would help a lot (:

**Comment by g-harel** [comment]
Alright, I've added them to the same folder

**Comment by solkimicreb** [comment]
Thanks, perfect! Super nice work (:
